flang/test/Driver/flang-linker-flags.f90

```diff
@@ -0,0 +1,17 @@
+! Verify that the Fortran runtime libraries are present in the linker
+! invocation. These libraries are added on top of other standard runtime
+! libraries that the Clang driver will include.
+
+! NOTE: The additional linker flags tested here are currently specified in
+! clang/lib/Driver/Toolchains/Gnu.cpp. This makes the current implementation GNU
+! (Linux) specific. The following line will make sure that this test is skipped
+! on Windows. Ideally we should find a more robust way of testing this.
+! REQUIRES: shell
+
+! RUN: %flang -### %S/Inputs/hello.f90 2>&1 | FileCheck %s
+
+! CHECK-LABEL: "/usr/bin/ld"
```

Comment:
I'd either keep it or remove completely. This is likely to match anything:
```
! CHECK-LABEL: "ld"
```

No strong opinion. I tested on AArch64 and X86 (Linux) and AFAIK this is the default location for the system linker. But the user could be using some other linker instead. It's probably safe to remove this line.

Reply:
I think it might be better to check for `ld` alone and then use `CHECK-SAME `to check the other libraries. While` ld` might individually match with anything, the `CHECK-SAME` looking for libraries hopefully saves the day. WDYT?

Reply:
Thanks for the suggestion!

> While ld might individually match with anything, the CHECK-SAME looking for libraries hopefully saves the day. WDYT?

Sadly, that didn't work :( I have another idea that leverages the fact that `flang-new` is implemented in terms of `clangDriver` and inherits all the options, e.g. [-ld-path](https://github.com/llvm/llvm-project/blob/9534e361ea12aaecde52b8ac4c947f9a301d0c9b/clang/include/clang/Driver/Options.td#L4400). Patch coming soon!

Reply:
That looks better.